.github/labeler.yml

```
@@ -0,0 +1,100 @@
+"area: Modem":
+  - "drivers/modem/**/*"
+"area: PWM":
+  - "drivers/pwm/**/*"
+"area: Watchdog":
+  - "drivers/watchdog/**/*"
+"area: Sensors":
+  - "drivers/sensors/**/*"
+"area: ADC":
+  - "drivers/adc/**/*"
+"area: Counter":
+  - "drivers/counter/**/*"
+"area: Timer":
+  - "drivers/timer/**/*"
+"area: I2S":
+  - "drivers/i2s/**/*"
+"area: C Library":
+  - "lib/libc/**/*"
+"area: Deviec Tree":
+  - "dts/**/*"
+  - "**/*.dts"
+"area: I2C":
+  - "drivers/i2c/**/*"
+"area: SPI":
+  - "drivers/spi/**/*"
+"area: Boards":
+  - "boards/**/*"
+"area: POSIX":
+  - "lib/posix/**/*"
+"area: native port":
+  - "arch/posix/**/*"
+  - "soc/posix/**/*"
+  - "**/*native_posix*"
+"area: X86":
+  - "arch/x86/**/*"
+"area: ARM":
+  - "arch/arm/**/*"
```

Should add the corresponding arch include dirs for all arch labels
```suggestion
"area: ARM":
  - "arch/arm/**/*"
  - "include/arch/arm/**/*"
```